tools/build_cros_gcc

```diff
@@ -105,19 +108,28 @@
    exit 0
  fi
 
- $0 -c
+ rm -rf \
+ ./$GLIBC \
+ ./$GMP \
+ ./$GCC \
+ ./$BINUTILS \
+ ./linux-$KERNEL_VERSION \
+ ./$MPFR \
+ ./$CLOOG \
+ ./$ISL \
+ ./$MPC
 
- echo $1 $2 $3
+ echo $0 $1 $2 $3
```

Comment:
This assumes all arguments are provided.  Aren't `arch_type` and `prefix` optional?

Reply:
Yes, they are. The script executes itself to fill in the arguments.

Reply:
This is also jut for debug. I can remove it.